src/api/model/body_model.py

```diff
@@ -49,19 +50,20 @@ def __init__(self):
                 .sort_values("count", ascending=False)
         self.graph = nx.from_pandas_edgelist(result, 'SOURCE_SUBREDDIT', 'TARGET_SUBREDDIT', create_using=nx.DiGraph())
 
+        self.sentimentboxhelper = SentimentBoxHelper(self.data)
+        self.MIN_SENTIMENT, self.MAX_SENTIMENT = self.sentimentboxhelper.run()
```

Comment:
I meant move these constants into the `SentimentBoxHelper` as they are thightly coupled to that functionality I would say. Then, whenever a user of `SentimentBoxHelper` wishes to obtain those properties, one can simply do

```python
sentboxhelper = SentimentBoxHelper(data)
sentboxhelper.MAX_SENTIMENT
```